### PR TITLE
Add darp6

### PR DESCRIPTION
**Pull request by jacobgkau**
This PR adds the tech specs, external overview, internal overview, and service manual for the Darter Pro 6.

### CODE REVIEW

File: src/models/darp6/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Darter Pro (darp6)
+
+- [External Overview](./external-overview.md)
+- [Internal Overview](./internal-overview.md)
+- [Parts & Repairs](./repairs.md)
+
+![Bonobo WS](./img/darp6.png)
```
Comment (crawfxrd) [inline src/models/darp6/README.md:7]: alt text

Comment (jacobgkau) [reply src/models/darp6/README.md]: Corrected with https://github.com/system76/tech-docs/pull/31/commits/9e2fb1fbd50b1492e1c41c07ca2c6b4be8bcf641, thank you.

**Review by ahoneybun — CHANGES_REQUESTED** [review]
It looks like the image path is .img on the md but the folder itself is just called img which is breaking the images on a local build.

**Comment by jacobgkau** [comment]
> It looks like the image path is .img on the md but the folder itself is just called img which is breaking the images on a local build.

@ahoneybun The images are working for me after cloning on a fresh machine. I do not see ".img" anywhere in the markdown, I see "./img" because it's a relative path (which all of the prior images are also using.) Do you have Git LFS set up on your system?

**Comment by ahoneybun** [comment]
I now have gti-lfs installed and have recloned the repo a few times but I'm missing images for a lot of items which work on the live version so I think it is just something on my end. I'm going to change my review now.

**Review by ahoneybun — APPROVED** [review]
LGTM.

**Review by jackpot51 — APPROVED** [review]

**Pull request merged**